lib/api/src/index.tsx

```diff
@@ -178,15 +179,17 @@ class ManagerProvider extends Component<ManagerProviderProps, State> {
       initStories,
       initURL,
       initVersions,
-    ].map(initModule => initModule({ ...routeData, ...apiData, state: this.state }));
+    ].map(initModule =>
+      initModule({ ...routeData, ...apiData, state: this.state, fullAPI: this.api })
+    );
```

Comment:
@tmeasday This is the change in API you needed before to gain access to fullAPI in the modules subAPI.